paymentsheet-example/src/main/java/com/stripe/android/paymentsheet/example/MainActivity.kt

```diff
@@ -107,6 +127,35 @@ private fun MenuItemRow(item: MenuItem) {
             fontWeight = FontWeight.SemiBold,
             modifier = Modifier.padding(bottom = 4.dp),
         )
+
         Text(text = item.subtitle)
+
+        if (item.isBeta) {
+            Chip(
+                colors = ChipDefaults.chipColors(
+                    backgroundColor = MaterialTheme.colors.secondary.copy(
+                        alpha = SurfaceOverlayOpacity,
+                    ),
+                ),
+                leadingIcon = {
+                    Icon(
+                        imageVector = Icons.Default.Info,
+                        contentDescription = null,
+                    )
+                },
+                onClick = context::openDecouplingBetaLink,
```

Comment:
hmm, this is like part generic yet part specific. Should we pass in the click handler to the menu item too? Or just deal with the inconsistency?

Reply:
Happy to tweak it!